engine/class_modules/priest/sc_priest.cpp

```diff
@@ -1110,8 +1112,13 @@ struct void_lasher_mind_sear_tick_t final : public priest_pet_spell_t
   {
     priest_pet_spell_t::impact( s );
 
-    p().o().generate_insanity( void_lasher_insanity, p().o().gains.insanity_eternal_call_to_the_void_mind_sear,
-                               s->action );
+    // Currently bugged to only give insanity on the main target
+    // https://github.com/SimCMinMax/WoW-BugTracker/issues/687
+    if ( s->target == main_target || !p().o().bugs )
```

Comment:
I *think* you can replace this with `s->target == parent_dot->target`

Reply:
looks great in testing that, thanks for the suggestion!